style/pattern_libraries/README.md

```diff
@@ -0,0 +1,31 @@
+Pattern Libraries
+=====
+
+What do we consider a "pattern libary" to be, exactly? At Sparkbox, our pattern libraries typically consist of all of the "components" that make up a project. We also usually combine those components into larger pieces or entire "pages" so that we have some kind of static represenation of how the components are used.
```

Comment:
Can we link to an external definition of a pattern library? A foundry article? Maybe one by Patrick Fulton? 😁 

https://seesparkbox.com/foundry/building_pattern_libraries_in_react_with_storybook